README.md

```diff
@@ -133,26 +133,26 @@ On a modern Linux host run the following:
 3. Use an S3 compatible client to create a bucket in the object store
 
    ```bash
-   s3cmd mb --no-ssl --host=127.0.0.1:53390 --host-bucket=  s3://rookbucket
+   s3cmd mb --no-ssl --host=${AWS_ENDPOINT} --host-bucket=  s3://rookbucket
```

Comment:
yes, we've been outputting an export statement for that var the whole time too, and it just makes the demo steps that much easier to cut/paste for multiple environments if that env var is included in the steps too